pkg/sql/privilege/privilege.go

```diff
@@ -29,20 +31,42 @@ type Kind uint32
 const (
 	_ Kind = iota
 	ALL
+	// ALTER privilege is only used for types. Since CRDB does not have the
+	// concept of ownership, ALTER privilege is required to ALTER the type.
+	// Having ALTER + GRANT act as ownership for a type.
+	ALTER
```

Comment:
Why do we need a new one here? What is the corresponding table permission to ALTER a table?

Reply:
The corresponding privilege for ALTER TABLE would be CREATE. I guess we could use CREATE for both for consistency since we already have precedent there.